src/Illuminate/Cache/Repository.php

```diff
@@ -373,7 +373,7 @@ public function forever($key, $value)
      * Get an item from the cache, or execute the given Closure and store the result.
      *
      * @param  string  $key
-     * @param  \DateTimeInterface|\DateInterval|int|null  $ttl
+     * @param  \Closure|\DateTimeInterface|\DateInterval|int|null  $ttl
```

Comment:
PR description says callable, but this is not a callable. this is more specific, a closure, not all callables are closures. all closures are callable.

Reply:
Excuse my wording. I guess Closure is more appropriate.